pkgs/tools/misc/bottom-rs/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, fetchFromGitHub, }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bottom-rs";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "bottom-software-foundation";
+    repo = pname;
+    rev = "3451cdadd7c4e64fe8e7f43e986a18628a741dec";
```

Comment:
See https://nixos.org/manual/nixpkgs/stable/#sec-package-naming:
`If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".`